barretenberg/cpp/src/barretenberg/dsl/acir_proofs/acir_composer.cpp

```diff
@@ -137,6 +187,14 @@ bool AcirComposer::verify_proof(std::vector<uint8_t> const& proof, bool is_recur
     }
 }
 
+bool AcirComposer::verify_proof_splitted(std::vector<uint8_t> const& public_inputs,
```

Comment:
These splitted methods are just here temporarily, they will become the main methods after I've checked everything